src/products/mozilla_org/testing.md

```diff
@@ -58,65 +58,67 @@ Updated translations are pushed to the production server almost daily.
 
-In some cases pages receive major updates that require a complete rewrite of the template: if this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.
+In some cases, pages receive major updates that require a complete rewrite of the template. If this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.
 
 ### Sync and update frequencies
 
 * Pontoon syncs every 20 minutes to the repository.
-* Web Dashboard and the Dev server update every 15 minutes.
+* The Dev server updates every 15 minutes.
 
-If you work on Pontoon, it is safe to say that it will take less than an hour to see your changes reflected on the dashboard and the dev server.
+If you work on Pontoon, it is safe to say that it will take less than an hour to see your changes reflected on the dev server.
 
-When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects.
+When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects. You can also check out the deadline at page level in Pontoon.
 
 ## Testing dynamic pages
 
 This section focuses on instructions for testing pages with dynamically generated content. Each page or topic is different in terms of steps or flow. These instructions could change over time to reflect product design updates. Linguistic testing is the main focus. The instructions below are detailed steps to get to the localized content so it can be reviewed in context.
 
-### [firefox/desktop/tips.lang](https://www.mozilla.org/firefox/desktop/tips/)
+### [firefox/accounts-2019.lang](https://www.mozilla.org/firefox/accounts/)
 
-* Click on the **next** or **back** link to check out the tips on different features.
-* Highlighted text is the same as in English.
-* Text fits inside the box.
+* Click the **Sign In to Monitor** button, you will be taken to the sign-in page in order to get to the [Firefox Monitor](https://monitor.firefox.com) site.
+* Click on the links under each of the following products:
+   * Firefox browser
+   * Firefox Lockwise
+   * Firefox Monitor
+   * Firefox Send
 
-### [firefox/new/horizon.lang](https://www.mozilla.org/firefox/new/)
+Note: You may see different languages between mozilla.org, the login window, and the products above. If any of these products is not localized in your locale, it will fall back to English. Or it will fall back to your second language preference if the product is translated into that language.
```

Comment:
your second language…into that language

I would talk about more than one language, i.e.

> If any of these products is not localized in your locale, it will fall back to other languages set as preferred for content language negotiation, assuming any of them is available for the project, or English."